### ISSUE TITLE
relay-compiler v13 excludes config property is not working properly

### ISSUE THREAD

**Issue by jcolakk** [facebook/relay#3721]
Hi 👋  firstly, wanna congratulate on v13 release 🎉 

Second, I'm having some troubles making `excludes` property in `relay.config.js` work. 

This is my config

```javascript
module.exports = {
  // Configuration options accepted by the `relay-compiler` command-line tool and `babel-plugin-relay`.
  src: './src',
  schema: './schema.graphql',
  schemaExtensions: ['./', './src/frontend'],
  language: 'typescript',
  customScalars: {
    DateTime: 'string',
    Date: 'string',
    Decimal: 'number',
  },
  excludes: ['/node_modules/', '/mocks/', '/generated/', '/backend/'],
};
```

I wanna exclude `backend` directory which is under `./src`. Our structure looks like this
```
src
  /backend
  /frontend
```

Compiler is not excluding it and it is throwing errors because we have some `.graphql` files under `backend`

Any idea what I might be doing wrong?

**Comment by alunyov** [comment]
Hmm, this is strange.

Can you try updating `schemaExtensions: ['./', './src/frontend'],` to include only directories where you define client schema extensions for your app. Also, I think, the correct format is `**/backend/**`.

**Comment by jcolakk** [comment]
Removing the `./` in `schemaExtensions` did the trick, seems it takes precedence over `excludes`, not sure if that is intentional. 

Also while I'm here, after running the compiler I get a lot of changed .graphql files, is that expected with the new compiler?

**Comment by alunyov** [comment]
> I get a lot of changed .graphql files, is that expected with the new compiler?

Yes, we've change the flow-type/typescript generation output. Also there some formatting changes.

**Comment by jcolakk** [comment]
Thanks a lot for the help ❤️  works like a charm
